<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutPersonne</class>
 <widget class="QDialog" name="AjoutPersonne">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>416</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl_warning">
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_age">
        <property name="text">
         <string>âge</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtbox_age"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_sexe">
        <property name="text">
         <string>sexe : </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="rdb_homme">
          <property name="text">
           <string>homme</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdb_femme">
          <property name="text">
           <string>femme</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbl_etat_civil">
        <property name="text">
         <string>état civil : </string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cb_etat_civil">
        <item>
         <property name="text">
          <string>marié</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>veuf</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>célibataire</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>vie maritale</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbl_compo_foyer">
        <property name="text">
         <string>composition du foyer : </string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_nb_personnes">
          <property name="text">
           <string>nombre de personnes : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spbox_nb_personnes"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbl_enfants">
          <property name="text">
           <string>dont enfants : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spbox_enfants"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lbl_animaux">
          <property name="text">
           <string>dont animaux domestiques : </string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spbox_animaux"/>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lbl_revenu">
        <property name="text">
         <string>tranche de revenu</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="cb_revenu">
        <item>
         <property name="text">
          <string>de 10 000 € à 25 000 €</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>de 25 000 € à 70 000 €</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>de 70 000 € à 150 000 €</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+ de 150 000 €</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
